Mathlib/FieldTheory/Normal.lean

```diff
@@ -359,6 +359,43 @@ theorem AlgEquiv.restrictNormalHom_surjective [Normal F K₁] [Normal F E] :
     Function.Surjective (AlgEquiv.restrictNormalHom K₁ : (E ≃ₐ[F] E) → K₁ ≃ₐ[F] K₁) := fun χ =>
   ⟨χ.liftNormal E, χ.restrict_liftNormal E⟩
 
+@[simp]
+theorem AlgEquiv.restrictNormalHom_id (F K : Type*)
+    [Field F] [Field K] [Algebra F K] [Normal F K] :
+    AlgEquiv.restrictNormalHom (F := F) (K₁ := K) K = MonoidHom.id (K ≃ₐ[F] K) := by
```

Comment:
Why `MonoidHom`? Can't you upgrade it to an `AlgebraHom`?

Reply:
I think you didn't get it correctly, the left side is the `MonoidHom` given by the restriction of the Galois group to a normal subextension, the right side is the identity `MonoidHom` to the smaller Galois group itself.

Reply:
Oh, I see. I am sorry, please forget this comment.